JitsiConference.js

```diff
@@ -475,7 +468,7 @@ JitsiConference.prototype._init = function(options = {}) {
             callStatsID: config.callStatsID,
             callStatsSecret: config.callStatsSecret,
             callStatsApplicationLogsDisabled: config.callStatsApplicationLogsDisabled,
-            enableCallStats,
+            enableCallStats: true,
```

Comment:
Can we drop this now, when it is always true? It was used to selectively enable/disable integration even when configured correctly. If we don't need it let it 🔥

Reply:
Done